### PR TITLE
Address review comments from #66262

### PR DESCRIPTION
**Pull request by metamben**
### Description

Just a small cleanup after #66262. (That PR was merged so as not to block other things.)

### How to verify

The tests should pass. There should be no change in behavior.

### CODE REVIEW

File: enterprise/backend/test/metabase_enterprise/data_studio/permissions/query_test.clj [hunk 1]
```diff
@@ -14,41 +14,40 @@
...
-                    (perms/revoke-collection-permissions! group-id collection-id)
-                    (perms/revoke-collection-permissions! (perms/all-users-group) collection-id)))
-                (binding [*current-user-id*              user-id
-                          *current-user-permissions-set* (atom (if collection-readable?
-                                                                 #{(perms/collection-read-path collection-id)}
-                                                                 #{}))]
-                  (testing (str "with features " (pr-str features)
-                                ", collection-readable? " collection-readable?
-                                ", view-data " view-data)
+      (testing (format "with features %s, collection-readable? %s, view-data %s"
```
Comment (galdre) [inline enterprise/backend/test/metabase_enterprise/data_studio/permissions/query_test.clj:21]: Thanks! The original placement of this was far enough down I didn't even notice it. I think the new placement is more readable and in edge cases (exceptions) could be slightly better.

**Review by galdre — APPROVED** [review]
